app/controllers/home_controller.rb

```diff
@@ -3,10 +3,10 @@ def about
   end
 
   def index
-    @posts = Post.published
+    @posts = Post.published.order(:published_at)
```

Comment:
What about creating a scope for this in the model and just use `Post.published` here?